src/resources/views/crud/fields/relationship/relationship_select.blade.php

```diff
@@ -50,7 +50,7 @@
 
 @include('crud::fields.inc.wrapper_start')
     <label>{!! $field['label'] !!}</label>
-
+    <input type="hidden" name="{{ $field['name'] }}" value="" @if(in_array('disabled', $field['attributes'] ?? [])) disabled @endif />
```

Comment:
@pxpm could you please explain what this does? I see it's been added in 3 fields but can't tell what it does and why it exists.

Reply:
I explained in comment: https://github.com/Laravel-Backpack/CRUD/pull/4016#discussion_r771464955